docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -55,43 +74,43 @@ these options:
 #### Using `cert-manager`
 
 You can use `cert-manager` to provision and automatically renew TLS credentials
-by completing ACME challenges via Let's Encrypt. We recommend this approach if
-you require CLI access to web applications using client certificates via
-the Teleport Application Service.
+by completing ACME challenges via Let's Encrypt.
 
-#### Using AWS Certificate Manager
+This method uses a Kubernetes `LoadBalancer`, which will provision an underlying AWS Network Load
+Balancer (NLB) to handle incoming traffic.
 
-You can use AWS Certificate Manager to handle TLS termination with AWS-managed
-certificates.
+#### Using AWS Certificate Manager
 
-You should be aware of the limitations for using AWS Certificate Manager to
-provision TLS credentials for Teleport:
+You can use AWS Certificate Manager to handle TLS termination with AWS-managed certificates.
 
-- This will prevent the Teleport Application Service from working via CLI using
-  client certificates. Application access will still work via a browser.
-- Command-line application access does not work with ACM. Using ACM will prevent
-  Teleport from facilitating application access via CLI (using client
-  certificates), as Teleport will not be handling its own TLS termination.
-- Using ACM through a AWS Load Balancer prevents the required traffic for
-  Postgres or MongoDB through Teleport's web port. If you choose to use the ACM
-  approach, we will show you how to configure a separate listener for Postgres
-  or MongoDB.
+This method uses a Kubernetes `Ingress`, which can provision an underlying AWS Application Load
+Balancer (ALB) to handle incoming traffic if one does not already exist. It also requires the
+installation and setup of the AWS Load Balancer controller.
 
-<Notice type="warning">
+You should be aware of these potential limitations when using Layer 7 load balancers with Teleport:
```

Comment:
Can we add a small description of why we need the proxy? 
At this point, it's not clear why Teleport enforces such limitations.
Explaining that Teleport uses x.509 certificates for authentication, we need that the client connection terminates in the Teleport proxy and that doesn't happen with L7...